src/scripts/browserDeviceProfile.js

```diff
@@ -413,38 +436,54 @@ import browser from './browser';
 
         if (canPlayAudioFormat('opus')) {
             videoAudioCodecs.push('opus');
-            hlsVideoAudioCodecs.push('opus');
+            hlsInTsVideoAudioCodecs.push('opus');
             webmAudioCodecs.push('opus');
         }
 
         if (canPlayAudioFormat('flac')) {
             videoAudioCodecs.push('flac');
+            hlsInFmp4VideoAudioCodecs.push('flac');
+        }
+
+        if (canPlayAudioFormat('alac')) {
+            videoAudioCodecs.push('alac');
+            hlsInFmp4VideoAudioCodecs.push('alac');
         }
 
         videoAudioCodecs = videoAudioCodecs.filter(function (c) {
             return (options.disableVideoAudioCodecs || []).indexOf(c) === -1;
         });
 
-        hlsVideoAudioCodecs = hlsVideoAudioCodecs.filter(function (c) {
+        hlsInTsVideoAudioCodecs = hlsInTsVideoAudioCodecs.filter(function (c) {
+            return (options.disableHlsVideoAudioCodecs || []).indexOf(c) === -1;
+        });
+
+        hlsInFmp4VideoAudioCodecs = hlsInFmp4VideoAudioCodecs.filter(function (c) {
             return (options.disableHlsVideoAudioCodecs || []).indexOf(c) === -1;
         });
 
         const mp4VideoCodecs = [];
         const webmVideoCodecs = [];
-        const hlsVideoCodecs = [];
+        const hlsInTsVideoCodecs = [];
+        const hlsInFmp4VideoCodecs = [];
+
+        if (browser.safari || browser.tizen || browser.web0s && canPlayHevc(videoTestElement, options)) {
```

Comment:
Shouldn't it be like this?
```suggestion
        if ((browser.safari || browser.tizen || browser.web0s) && canPlayHevc(videoTestElement, options)) {
```
Otherwise, `canPlayHevc` is called only for webOS.

Reply:
oops!